src/content/docs/en/guides/imports.mdx

```diff
@@ -200,6 +200,11 @@ Additionally, glob patterns must begin with one of the following:
  
 
 [Read more about the glob pattern syntax](https://github.com/mrmlnc/fast-glob#pattern-syntax).
+
+#### Astro.glob() vs Content Collections
+
+[Content collections](https://docs.astro.build/en/guides/content-collections/) were introduced in `astro@2.0.0`. It's the best way to work with Markdown and MDX files in an Astro project. If your directory contains only Markdown and MDX files, it's best to use Content collections and the `src/content` directory.
```

Comment:
Fix for the absolute link
```suggestion
#### Astro.glob() vs Content Collections

[Content collections](/en/guides/content-collections/) were introduced in `astro@2.0.0`. It's the best way to work with Markdown and MDX files in an Astro project. If your directory contains only Markdown and MDX files, it's best to use Content collections and the `src/content` directory.
```

Reply:
Aye good looking out @Yan-Thomas. Thanks!